services/monitoring/Makefile

```diff
@@ -17,63 +17,31 @@ TEMP_COMPOSE=.stack.${STACK_NAME}.yaml
 # - all services to monitor must be in MONITORED_NETWORK
 #
 
-# Network that includes all services to monitor
-#  - by default it will create an overal attachable network called monitored_network
-ifeq ($(monitored_network),)
-MONITORED_NETWORK = monitored_network
-else
-MONITORED_NETWORK := $(monitored_network)
-endif
 
 # postgres db ulr
 POSTGRES_EXPORTER_DATA_SOURCE_NAME ?= postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@postgres:5432/${POSTGRES_DB}?sslmode=disable
 
-export MONITORED_NETWORK
 export POSTGRES_EXPORTER_DATA_SOURCE_NAME
 
-# Network from which services are reverse-proxied
-#  - by default it will create an overal attachable network called public_network
-ifeq ($(public_network),)
-PUBLIC_NETWORK = public-network
-else
-PUBLIC_NETWORK := $(public_network)
-endif
-export PUBLIC_NETWORK
 
 # TARGETS --------------------------------------------------
 include $(realpath $(CURDIR)/../../scripts/common.mk)
 
+# Makefile including function used by every services
+include ../common.Makefile
```

Comment:
common is definitively making every Makefile more readable and making easier to spot differences